### PR TITLE
Alpine dependencies installation script

### PR DESCRIPTION
**Pull request by 0rzech**

### CODE REVIEW

File: install_alpine_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  curl \
+  llvm4 \
+  `# federation` \
+  libxml2-dev \
+  libxslt-dev \
+  python3-dev \
+  `# socialhome` \
+  git \
+  linux-headers \
+  nodejs \
```
Comment (jaywink) [inline install_alpine_dependencies.sh:49]: Which Node is this btw? Currently I think anything goes (though I have only tested 4+), but to compile the Vue.js frontend bundle, Node 6 seems to be required. I have been meaning to add this already to the documentation, even though the Vue frontends are not ready yet.

**Review by jaywink — COMMENTED** [review]
Just one comment re node version!

File: install_alpine_dependencies.sh (hunk 1 above)
Comment (0rzech) [reply install_alpine_dependencies.sh:49]: It's [this](https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.6) one.

Comment (jaywink) [reply install_alpine_dependencies.sh:49]: Thanks that is sufficient 👍

**Pull request merged**